### ISSUE TITLE
Causes issue with AMD Adrenalin.

### ISSUE THREAD

**Issue by KobeRiddle-dev** [itsmikethetech/Virtual-Display-Driver#164]
When the virtual display driver is installed, AMD Adrenalin will not launch. I uninstalled the display driver and AMD Adrenalin launched successfully. No other changes made to the system.

Hardware:
CPU - Ryzen 5700x
GPU - Radeon 6600
32 GB RAM

Windows 11

This is unfortunate, as using the virtual monitor for sunshine was a nice plus.

**Comment by itsmikethetech** [comment]
Interesting. I have not experienced this. I will test on an AMD GPU ASAP and see if I can recreate the issue. Latest Adrenaline?

**Comment by KobeRiddle-dev** [comment]
> Interesting. I have not experienced this. I will test on an AMD GPU ASAP and see if I can recreate the issue. Latest Adrenaline?

Yes, the latest adrenalin. I tried reinstalling adrenalin and uninstalling Sunshine as well.

**Comment by alxtraxxx** [comment]
For me it works with current drivers if not streaming but Adrenaline is really buggy with overlays. If I use a real physical headless plug or vdd I cant enable overlay (or at least it looks like), if I turn on main monitor then I can see overlay is drawn on wrong monitor. Didnt find a way yet to fix it, didnt try disabling other monitor completely in windows yet. Other overlays like steam/reshade/ etc. work so my guess is vdd can not fix that as it seems to be an AMD bug. Like adrenaline overlays wont work for me after energy saving mode too

**Comment by itsmikethetech** [comment]
I was not able to recreate the issue with loading. It seem to work fine in conjunction with the AMD Adrenaline software. HOWEVER, alxtraxxx is correct in that the overlay does not show on the virtual display. I was able to recreate this issue in my testing. Bud and I are looking into this, but unsure of an immediate fix. (Unsure if it's on AMD-side or if there's something we can do to enable the functionality.)

**Comment by alxtraxxx** [comment]
I am not sure if its correlated but I have no Tab to configure my displays in Adrenaline any more when VDD is installed so I cant switch pixel format etc. Havent tested if it will return when I uninstall VDD yet
